ReactAndroid/src/main/third-party/android/androidx/BUCK

```diff
@@ -32,6 +32,15 @@ fb_native.android_library(
     ],
 )
 
+fb_native.android_library(
+    name = "autofill",
+    visibility = ["PUBLIC"],
+    exported_deps = [
```

Comment:
@safaiyeh perhaps you can try to add `":annotation"` into `exported_deps` as well? I digged [autofill source code](https://android.googlesource.com/platform/frameworks/support/+/refs/heads/androidx-main/autofill/autofill/src/main/java/androidx/autofill/HintConstants.java) and found that they do imported `androidx.annotation` library.

(Not familiar with BUCK though, please ignore this comment if you know it's not causing issues)

Reply:
testing this now 🤞

Reply:
This was the solution appreciate it! Hopefully we can get this in now

Reply:
Unfortunately, this wasn't the fix. But tried some more BUCK changes and got it working with this commit https://github.com/facebook/react-native/pull/28008/commits/2cdcff33aff337be24774d7558e2d5e9728ff445 !